workspaces/arborist/lib/arborist/reify.js

```diff
@@ -123,10 +124,19 @@ module.exports = cls => class Reifier extends cls {
       // we do NOT want to set ownership on this folder, especially
       // recursively, because it can have other side effects to do that
       // in a project directory.  We just want to make it if it's missing.
-      await mkdir(resolve(this.path), { recursive: true })
+      const resolvedPath = resolve(this.path)
+      try {
+        await access(resolvedPath)
+      } catch (accessError) {
+        if (accessError.code === 'ENOENT') {
+          await mkdir(resolvedPath, { recursive: true })
+        } else {
+          throw accessError
```

Comment:
The reason tests are failing is because there aren't any that test for this scenario, i.e. an error other than `ENOENT` happening.

Reply:
Will any changes have to be made to this code to resolve the failing tests in this case?

Reply:
A test will have to be added to cover that code path.